### PR TITLE
Docs/property requirements

### PR DESCRIPTION
**Pull request by cstruct**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,33 +36,47 @@ allowed, I'm only using them for demonstration purposes.)
 ```javascript
 {
 
-  // The title of the produced DMG, which will be shown when mounted
+  // The title of the produced DMG, which will be shown when mounted (Required)
   "title": "Test Title",
 
   // Path to your icon, which will be shown when mounted
```
Comment (LinusU) [inline README.md]: Should we add ` (Optional)` here?

Comment (cstruct) [inline README.md]: I think it feels a bit redundant.

File: README.md [hunk 2]
```diff
@@ -36,33 +36,47 @@ allowed, I'm only using them for demonstration purposes.)
...
-  // Size of all the icons inside the DMG
+  // Size of all the icons inside the DMG (Required)
   "icon-size": 80,
   
   // Window options
   "window": {
     // Position when opened
-    "position": { "x": 250, "y": 250 },
+    "position": {
+      // X position relative to bottom of the screen (Required)
```
Comment (LinusU) [inline README.md]: **X** position relative to **bottom**?

File: README.md (hunk 1 above)
Comment (LinusU) [inline README.md]: 👌

**Comment by LinusU** [comment]
Doesn't it feel like the entire `window` object is required now though?

File: README.md (hunk 2 above)
Comment (cstruct) [inline README.md]: Haha facepalm

**Comment by LinusU** [comment]
> This branch has conflicts that must be resolved

Sorry about that 💃

**Comment by cstruct** [comment]
@LinusU, i changed the spec to MSON. What do you think?

**Comment by LinusU** [comment]
To be honest, I don't love it :)

Especially when viewing it as rendered markdown it doesn't look too good. Maybe we should have a "quick start" example in JSON, and then document all the options in detail further down?

**Comment by LinusU** [comment]
Minimal quick start example:

```js
{

  // The title of the produced DMG, which will be shown when mounted
  "title": "Test Title",

  // Path to your icon, which will be shown when mounted
  "icon": "TestIcon.icns",

  // Path to your background
  "background": "TestBkg.png",

  "contents": [

    // This is the contents of your DMG.

    // Each entry has a position specified by
    // X and Y in the center of its icon.

    // `type: link` creates a link to the specified target
    { "x": 448, "y": 344, "type": "link", "path": "/Applications" },

    // `type: file` adds a file to the DMG
    { "x": 192, "y": 344, "type": "file", "path": "TestApp.app" }

  ]

}
```

**Comment by cstruct** [comment]
@LinusU 
Updated to use a working JSON example and a simplified MSON (without value examples and all keys qouted).

**Comment by LinusU** [comment]
Looking awesome 🙌 LGTM

**Pull request merged**